guides/v2.3/install-gde/prereq/optional.md

```diff
@@ -31,7 +31,9 @@ See one of the following sections:
 
 Enter the following command to install NTP:
 
-	apt-get install ntp
+```bsh
```

Comment:
```suggestion
```bash
```

Reply:
Manually applied change to local branch since I'm making other local changes.